Source/BugsnagConfiguration.m

```diff
@@ -218,23 +218,6 @@ - (void)setApiKey:(NSString *)apiKey {
     }
 }
 
-@synthesize shouldAutoCaptureSessions = _shouldAutoCaptureSessions;
-
-- (BOOL)shouldAutoCaptureSessions {
-    return _shouldAutoCaptureSessions;
-}
-
-- (void)setShouldAutoCaptureSessions:(BOOL)shouldAutoCaptureSessions {
-    @synchronized (self) {
-        _shouldAutoCaptureSessions = shouldAutoCaptureSessions;
-
-        if (shouldAutoCaptureSessions) { // track any existing sessions
```

Comment:
IIRC this logic (or similar) may have been added to enable automatic session tracking on React Native. Have we tested the effects of these changes on downstream notifiers?

Reply:
bugsnag-react-native will need a slight tweak in the initialization logic. While configuring the JS options, when all of the following are true:

* Bugsnag has already been configured on the native side
* The native configuration has `shouldAutoCaptureSessions` set to `NO`
* The JS configuration has `autoCaptureSessions` set to `true`

Then `[BugsnagReactNative start]` should call `startSession`

Reply:
Yeh there wouldn't be an issue with it the way it is setup right now. For the native notifiers we turn off automatic session tracking